### ISSUE TITLE
hi~ one .pth file link  is out of date,Do you have this file locally? If so, can you upload it to Google Cloud disk?

### ISSUE THREAD

**Issue by Samchengjiaming** [lichengunc/mask-faster-rcnn#12]
![图片](https://user-images.githubusercontent.com/35476142/199023122-e2ca72da-8722-4756-b5c7-4aa5b6b8bd23.png)

bvision.cs.unc.edu/licheng/MattNet/pytorch_mask_rcnn/res101_mask_rcnn_iter_1250k.zip

**Comment by Buki2** [comment]
Thank you for your interest in our code repository.

Yes, we saved this file locally and uploaded it to Google Drive.
[Here's the link.](https://drive.google.com/file/d/1IjLQdd2JE3-kOE934fnU9V1jpmgdgiZW/view?usp=share_link)

There is an optional solution. You can download the pre-trained model of faster-rcnn from the link provided at https://github.com/ruotianluo/pytorch-faster-rcnn.
[Here's the link.](https://drive.google.com/open?id=0B7fNdx_jAqhtNE10TDZDbFRuU0E)

**Comment by zjj7715** [comment]
Has the issue been resolved? If you have this file locally，can you upload it ？
